calm_llm/data/flows/book_hotel.yml

```diff
@@ -27,4 +36,12 @@ flows:
         description: Name of the hotel which the user has selected. Must be an exact element of the slot "hotel_search_results_readable". If the user wants an option that isn't listed, do not fill the slot with that value.
       - collect: hotel_booking_confirmation
         ask_before_filling: true
-      - action: utter_confirm_booked_hotel
+      - noop: true
+        next:
```

Comment:
Yes, it can be. 
The reason I used this structure to be consistent with the structure that was used after the `collect: hotel_end_date` in the same flow.

I will change both to skip the `noop: true`

Reply:
This is now fixed